### PR TITLE
feature/is-nil

### LINKED ISSUE #25: Refactor `isPresent` and `isBlank` to use `isNil`
`isNil` is a thing in lodash 4+. `isPresent` should become the opposite of it and `isBlank` should just go away.

### PR DESCRIPTION
**Pull request by rjhilgefort**
Fixes #25 

Cleaned up tests a bit- also added some spacing opinions to tests.

@njutte please review since the tests were your baby.

cc @AO16

### CODE REVIEW

File: tests/lodash-deep-extras-tests.js [hunk 1]
```diff
@@ -289,232 +228,165 @@ describe(
```
Comment (njutte) [inline tests/lodash-deep-extras-tests.js:225]: Why did these guys get to keep their spacing?

**Comment by njutte** [comment]
One little thing but otherwise looks fine.

File: tests/lodash-deep-extras-tests.js (hunk 1 above)
Comment (rjhilgefort) [inline tests/lodash-deep-extras-tests.js:225]: Do you mean the space above `beforeEach`? I went with the following rule: new line above `describe` `beforeEach` `afterEach`... effectively, just no new line above `it`

Comment (njutte) [inline tests/lodash-deep-extras-tests.js:225]: I was referring to the 'isInteger' test it didn't get the updated spacing applied

Comment (rjhilgefort) [inline tests/lodash-deep-extras-tests.js:225]: Thanks, must have missed that.

**Pull request merged**